### ISSUE TITLE
Compatibility with Blender 4.x

### ISSUE THREAD

**Issue by testman42** [neverhood311/Stop-motion-OBJ#184]
Are there any plans to make this extension work with Blender 4 ?

**Comment by neverhood311** [comment]
Yes, I've been working on it but I've run into a hiccup with the refactor. Because of how they've written the new OBJ and PLY importers, I can't get the Streaming Sequence feature to work. I'm looking into my options for fixing this.

**Comment by dumoulina** [comment]
I did a fork that support blender 4. That's quick and dirty but it worked fine at some point.
I also added import sequences from zip file and removed the need of choosing the file extension for my own needs.
https://github.com/Tendocat/Stop-motion-OBJ/tree/blender4

**Comment by neverhood311** [comment]
@Tendocat Does it work when you import a streaming sequence and try to render an animation? For me, it tends to crash Blender 4.x after a few frames. Either that or it just stops loading the next frame.

**Comment by dumoulina** [comment]
No crash occured for now.
All loaded meshes are not deleted so you may still run out of memory.
It should not be that hard to clean up but it needs some reading 
of the blender API.

I attached my system info in case that is the issue. I'm in 4.2  
[system-info.txt](https://github.com/user-attachments/files/16443215/system-info.txt)

**Comment by neverhood311** [comment]
I've just released an update that works with Blender 4.x:
https://github.com/neverhood311/Stop-motion-OBJ/releases/tag/v2.3.0
